File: python/pattern/pat7.py
n = 5

for i in range(n):
    # spaces
    for j in range(n-i-1):
        print(" ",end="")
    
    # star 1
    for k in range(i+1):
        print("*", end="")
    
    # star 2
    for l in range(i):
        print("*",end="")
    
    print()

File: cpp/pattern/pat7.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n = 5;
    for(int i = 0; i<n;i++){
        // spaces
        for(int j = 0; j<n-i-1;j++){
            cout<<" ";
        }
        // star1
        for(int k = 0; k<i+1; k++){
            cout<<"*";
        }

        // star2
        for(int k = 0; k<i;k++){
            cout<<"*";
        }
        cout<<endl;
    }
    return 0;
}